### PR TITLE
homework to cypress commands

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Luibov,
Please review my pull request when you have a chance.
Thank you
Svitlana

### CODE REVIEW

File: cypress/e2e/app_tests/cypressInrtro.cy.js [hunk 1]
```diff
@@ -18,4 +18,46 @@ describe('second test suite', () => {
...
+                cy.visit('/pages/tables/smart-table');
+                cy.get('[placeholder="ID"]').type(12);
+                cy.wait(1000);
+                cy.get('tbody tr td').each(($el) => {
+                        let element1 = $el.text();
+                        element.push(element1);
+                        cy.log(element);
+                })
+        })
+        it.only('working with tables', () => {
```
Comment (landriiuk) [inline cypress/e2e/app_tests/cypressInrtro.cy.js]: Pls delete only here

File: cypress/e2e/app_tests/cypressInrtro.cy.js [hunk 2]
```diff
@@ -18,4 +18,46 @@ describe('second test suite', () => {
                 cy.visit('pages/modal-overlays/tooltip');
                 cy.contains('Show Tooltip').eq(0).realHover();
         })
-})
\ No newline at end of file
+})
+describe('Scrolling on the page', () => {
+        it('scrolling using scroll into view', () => {
+               // cy.visit('auth/login');
+                //cy.get('[aria-label="Register"]').scrollIntoView({duration: 500});
```
Comment (landriiuk) [inline cypress/e2e/app_tests/cypressInrtro.cy.js:25]: Is this code not working? Do you need any help with it?

File: cypress/e2e/app_tests/adding_row_in_table.cy.js
Comment (landriiuk) [inline cypress/e2e/app_tests/adding_row_in_table.cy.js:1]: You have added empty row here, would be better add row and fill it, just to practice a bit more

**Review by landriiuk — APPROVED** [review]
Overall looks good!
You can merge this pr, or rework that part i mentioned

File: cypress/e2e/app_tests/cypressInrtro.cy.js (hunk 1 above)
Comment (manatskaya) [reply cypress/e2e/app_tests/cypressInrtro.cy.js]: Thank you for review
updated

**Pull request merged**